core/src/filter/resources/jenkins/model/jenkins-version.properties

```diff
@@ -1 +1,2 @@
-version=${build.version}
\ No newline at end of file
+version=${build.version}
+java.min.level=${java.level}
```

Comment:
The file is a generic property file, so it should not really matter how many properties are there. I would also rework the logic, but I do not think it's something to be done in this PR. WDYT?